docker/labels.go

```diff
@@ -11,10 +11,11 @@ type Label string
 
 // Libcompose default labels.
 const (
-	NAME    = Label("io.docker.compose.name")
-	PROJECT = Label("io.docker.compose.project")
-	SERVICE = Label("io.docker.compose.service")
-	HASH    = Label("io.docker.compose.config-hash")
+	NAME     = Label("io.docker.compose.name")
+	PROJECT  = Label("io.docker.compose.project")
+	SERVICE  = Label("io.docker.compose.service")
+	HASH     = Label("io.docker.compose.config-hash")
+	RECREATE = Label("io.docker.compose.recreate")
```

Comment:
@dnephin Sorry there was a terrible typo in that last comment I just fixed.  I should have said "We can just remove the ability"...